modules/nextflow/src/main/groovy/nextflow/trace/TraceRecord.groovy

```diff
@@ -103,7 +103,8 @@ class TraceRecord implements Serializable {
             vol_ctxt: 'num',        // -- /proc/$pid/status field 'voluntary_ctxt_switches'
             inv_ctxt: 'num',        // -- /proc/$pid/status field 'nonvoluntary_ctxt_switches'
             hostname: 'str',
-            cpu_model:  'str'
+            cpu_model:  'str',
+            num_reclamations: 'num'
```

Comment:
TraceRecord structure should not be impacted by this. Use instead the same pattern for machine info metadata 

https://github.com/nextflow-io/nextflow/blob/458ef97a9f47e256da26a13c6dfa27ba2eb258c9/plugins/nf-tower/src/main/io/seqera/tower/plugin/TowerClient.groovy#L658-L662